WFInfo/Main.cs

```diff
@@ -101,40 +103,58 @@ public static void ThreadedDataLoad()
         }
         private static async void TimeoutCheck()
         {
-            if (!await dataBase.IsJWTvalid())
+            if (!await dataBase.IsJWTvalid().ConfigureAwait(true))
                 return;
-            var now = DateTime.UtcNow;
+            DateTime now = DateTime.UtcNow;
             Debug.WriteLine($"Checking if the user has been inactive \nNow: {now}, Lastactive: {latestActive}");
 
-            if (OCR.Warframe != null && OCR.Warframe.HasExited)
+            if (OCR.Warframe != null && OCR.Warframe.HasExited && LastMarketStatus != "invisible")
             {//set user offline if Warframe has closed but no new game was found
+                Debug.WriteLine($"Warframe was detected as closed");
+
                 await Task.Run(async () =>
                 {
-                    if (!await dataBase.IsJWTvalid())
+                    if (!await dataBase.IsJWTvalid().ConfigureAwait(true))
                         return;
-                    await dataBase.SetWebsocketStatus("invisible");
+                    //IDE0058 - computed value is never used.  Ever. Consider changing the return signature of SetWebsocketStatus to void instead
+                    await dataBase.SetWebsocketStatus("invisible").ConfigureAwait(false);
                     StatusUpdate("WFM status set offline, Warframe was closed", 0);
-                });
+                }).ConfigureAwait(false);
             }
-            
-            if (latestActive <= now)
+            else if (UserAway && latestActive > now)
```

Comment:
conditions are now else instead of all being simultaneously possible (which is nonsense)